conf/development-websocket.ini

```diff
@@ -2,6 +2,7 @@
 use: call:h.websocket:create_app
 
 # Elasticsearch configuration
+es.host: http://localhost:9200
```

Comment:
I don't think the WebSocket actually needs ES at all, but I've left removing that dependency as an exercise for a separate PR. Until then this setting is needed in a dev environment.